docs/performance.md

```diff
@@ -4,23 +4,19 @@ description: The performance difference between es-toolkit and alternative libra
 
 # Performance
 
-![Graph showing the difference in performance between es-toolkit and lodash. Using es-toolkit results in a performance gain of up to 11 times.](/assets/performance.png)
-
-Designed with performance in mind, es-toolkit provides an average of 2× performance improvement compared to alternative libraries like lodash. Some functions achieve up to an 11× performance gain by fully utilizing modern JavaScript APIs.
```

Comment:
Currently, the difference is not up to 11 times, so this part can be considered an exaggeration, so it is removed.